migration-guides/v4.0.0.md

```diff
@@ -0,0 +1,7 @@
+# v4.0.0 Migration Guide
+
+_Note: this guide assumes you are upgrading from decanter v3 to v4._
+
+This version contains the following breaking changes:
+
+1. `FloatParser` and `IntegerParser` have been updated to address a bug where negative numbers were being parsed as positive. In the (unlikely) event that your project was relying on the previous behavior, you can include the legacy version(s) of the parsers as custom parsers in your project. Their source code is available on the `v3` branch.
```

Comment:
@chawes13 i assume a v3 branch is created as part of the new version release process?

Reply:
At the moment I would just advise them to more specifically pin their version. Unless there was a huge security flaw found, we very likely would not continue to release updates to `<v4`, so a branch isn't necessary. If that edge case does occur, we can create that branch from the `v3.5.1` tagged commit.

Reply:
Could you add a small code snippet to demonstrate using one of the legacy parsers as a custom parser?

Reply:
@dianacamacho Just wanted to make sure you saw this. We're very close!

Reply:
@chawes13 updated